test/reshape.jl

```diff
@@ -654,4 +654,12 @@ end
     @test IndexStyle(DataFrames.StackedVector) == IndexLinear()
 end
 
+@testset "empty unstack" begin
+    df = DataFrame(a = [], b = [], c = [])
+    dfu = unstack(df, :b, :c)
+    @test isempty(dfu)
+    @test names(dfu) == ["a"]
+    @test dfu.a isa Vector{Any}
```

Comment:
Any particular reason to check this type?

Reply:
For current and future safety I guess. I was surprised that:
```
julia> Symbol.(Any[])
Any[]
```
because I knew that:
```
julia> map(Symbol, Any[])
Symbol[]
```
so I thought it does not hurt to be extra careful with empty collections and add a test.